busted-scm-2.rockspec

```diff
@@ -30,6 +38,14 @@ dependencies = {
   'mediator_lua >= 1.1.1',
 }
 
+test_dependencies = {
+  "busted",
```

Comment:
curently also luacov, unless it is removed from `.busted`

Reply:
c.f. https://github.com/lunarmodules/busted/pull/689#discussion_r952411360